src/components/Tool/Appeal/AppealsContext/AppealsContext.tsx

```diff
@@ -262,22 +262,30 @@ export const AppealsProvider: React.FC<AppealsContextProps> = ({
     debounce((searchTerm: string) => {
       const { searchTerm: _, ...oldQuery } = query;
       if (searchTerm !== '') {
-        replace({
-          pathname,
-          query: {
-            ...oldQuery,
-            accountListId,
-            ...(searchTerm && { searchTerm }),
+        replace(
+          {
+            pathname,
+            query: {
+              ...oldQuery,
+              accountListId,
+              ...(searchTerm && { searchTerm }),
+            },
           },
-        });
+          undefined,
+          { shallow: true },
```

Comment:
I'm still having an issue when searching. When I type something into the search bar it takes me back to the list of all the appeals.

Reply:
This will be fixed on the `MPDX-7948-creating-appeal-details-pages-2`; I separated the initial & detail appeals pages into separate pages.